tools/dashboard-plugin-template/README.template.md

```diff
@@ -52,7 +52,7 @@ Building a new plugin is easy. Before you start, ensure that your `.env` and `.e
     Note: we recommend to write your plugin in TypeScript and to use a modern IDE. This way you can conveniently
     explore the plugin customization APIs from the comfort of your development environment.
 
-3.  Build the plugin: `{{packageManager}} build-plugin`
+3.  Build the plugin: `{{packageManager}} run build-plugin`
```

Comment:
will this work in npm?

Reply:
More so, the version without run would only work with yarn. All the other {{packageManager}} occurrences have the run, only this one missed it.

Reply:
doh, i misread.. i thought you were removing the `run`.. eh